odrive/torque_test/m12.json

```diff
@@ -0,0 +1 @@
e": {"config": {"gpio_num": 0, "is_active_low": true}}, "min_endstop": {"config": {"debounce_ms": 50, "enabled": false, "gpio_num": 0, "is_active_high": false, "offset": 0.0}}, "motor": {"config": {"I_bus_hard_max": Infinity, "I_bus_hard_min": -Infinity, "I_leak_max": 0.10000000149011612, "R_wL_FF_enable": false, "acim_autoflux_attack_gain": 10.0, "acim_autoflux_decay_gain": 1.0, "acim_autoflux_enable": false, "acim_autoflux_min_Id": 10.0, "acim_gain_min_flux": 10.0, "bEMF_FF_enable": false, "calibration_current": 10.0, "current_control_bandwidth": 100.0, "current_lim_margin": 8.0, "current_lim": 20.0, "dc_calib_tau": 0.20000000298023224, "inverter_temp_limit_lower": 100.0, "inverter_temp_limit_upper": 120.0, "motor_type": 0, "phase_inductance": 0.00018064251344185323, "phase_resistance": 0.11777061223983765, "pole_pairs": 15, "pre_calibrated": true, "requested_current_range": 25.0, "resistance_calib_max_voltage": 4.0, "torque_constant": 0.6778688430786133, "torque_lim": Infinity}, "fet_thermistor": {"config": {"enabled": true, "temp_limit_lower": 100.0, "temp_limit_upper": 120.0}}, "motor_thermistor": {"config": {"enabled": false, "gpio_pin": 4, "poly_coefficient_0": 0.0, "poly_coefficient_1": 0.0, "poly_coefficient_2": 0.0, "poly_coefficient_3": 0.0, "temp_limit_lower": 100.0, "temp_limit_upper": 120.0}}}, "sensorless_estimator": {"config": {"observer_gain": 1000.0, "pll_bandwidth": 1000.0, "pm_flux_linkage": 0.0015800000401213765}}, "trap_traj": {"config": {"accel_limit": 0.5, "decel_limit": 0.5, "vel_limit": 2.0}}}, "can": {"config": {"baud_rate": 250000, "protocol": 1}}, "config": {"brake_resistance": 2.0, "dc_bus_overvoltage_ramp_end": 59.92000198364258, "dc_bus_overvoltage_ramp_start": 59.92000198364258, "dc_bus_overvoltage_trip_level": 59.92000198364258, "dc_bus_undervoltage_trip_level": 8.0, "dc_max_negative_current": -0.009999999776482582, "dc_max_positive_current": Infinity, "enable_brake_resistor": true, "enable_can_a": true, "enable_dc_bus_overvoltage_ramp": false, "enable_i2c_a": false, "enable_uart_a": true, "enable_uart_b": false, "enable_uart_c": false, "error_gpio_pin": 0, "gpio10_mode": 11, "gpio11_mode": 2, "gpio12_mode": 12, "gpio13_mode": 12, "gpio14_mode": 2, "gpio15_mode": 7, "gpio16_mode": 7, "gpio1_mode": 4, "gpio2_mode": 4, "gpio3_mode": 3, "gpio4_mode": 3, "gpio5_mode": 3, "gpio6_mode": 0, "gpio7_mode": 0, "gpio8_mode": 0, "gpio9_mode": 11, "max_regen_current": 0.0, "uart0_protocol": 3, "uart1_protocol": 3, "uart2_protocol": 3, "uart_a_baudrate": 115200, "uart_b_baudrate": 115200, "uart_c_baudrate": 115200, "usb_cdc_protocol": 3, "gpio1_pwm_mapping": {"endpoint": null, "max": 0.0, "min": 0.0}, "gpio2_pwm_mapping": {"endpoint": null, "max": 0.0, "min": 0.0}, "gpio3_analog_mapping": {"endpoint": null, "max": 0.0, "min": 0.0}, "gpio3_pwm_mapping": {"endpoint": null, "max": 0.0, "min": 0.0}, "gpio4_analog_mapping": {"endpoint": null, "max": 0.0, "min": 0.0}, "gpio4_pwm_mapping": {"endpoint": null, "max": 0.0, "min": 0.0}}}
```

Comment:
Pls autoformat so that it's readable and not a single line.

Reply:
no u

Reply:
Done in f52cba8